src/features/governance/components/ProposalVotesHistoryTable.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from 'next/link';
+import { FullWidthSpinner } from 'src/components/animation/Spinner';
+import { SolidButton } from 'src/components/buttons/SolidButton';
+import { HeaderAndSubheader } from 'src/components/layout/HeaderAndSubheader';
+import { PersonalizedProposalCard } from 'src/features/account/PersonalizedProposalCard';
+import { useDelegateeHistory } from 'src/features/delegation/hooks/useDelegateeHistory';
+import { useGovernanceProposals } from 'src/features/governance/hooks/useGovernanceProposals';
+import { objLength } from 'src/utils/objects';
+import { useAccount } from 'wagmi';
+
+export function ProposalVotesHistoryTable() {
+  const account = useAccount();
+  const { proposalToVotes, isLoading: isLoadingHistory } = useDelegateeHistory(account.address!);
+  const { proposals, isLoading: isLoadingProposals } = useGovernanceProposals();
+  const isLoading = isLoadingHistory || isLoadingProposals;
+
+  if (isLoading) {
+    return <FullWidthSpinner>Loading historical data</FullWidthSpinner>;
+  }
+
+  if (!objLength(proposalToVotes!)) {
+    return (
+      <HeaderAndSubheader
+        header="No history available"
+        subHeader={`You don’t currently have any recorded governance votes with the logged in address. Delegated votes won't show.`}
```

Comment:
```suggestion
        subHeader={`You don’t currently have any recorded governance votes with the connected account. Delegated votes won't show.`}

```

or "logged-in" but i think "connected" sounds better and is more accurate